### PR TITLE
Fixes 550: Implement app invite via facebook

### PR DESCRIPTION
**Pull request by rohanagarwal94**
Fixes issue #550 

Changes: Now app invite can be sent through facebook.

Screenshots for the change: 
screenshot_2017-02-23-02-04-49-745_org fossasia susi ai

### CODE REVIEW

File: app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java [hunk 1]
```diff
@@ -1417,18 +1431,77 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
+                                            .build();
+                                    AppInviteDialog appInviteDialog = new AppInviteDialog(MainActivity.this);
+                                    appInviteDialog.registerCallback(callbackManager, new FacebookCallback<AppInviteDialog.Result>() {
+                                        @Override
+                                        public void onSuccess(AppInviteDialog.Result result) {
+                                            alertDialog.cancel();
+                                        }
+
+                                        @Override
+                                        public void onCancel() {
```
Comment (rohanagarwal94) [inline app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java]: @jig08 @the-dagger @daminisatya  should the method be left empty or should I include a Log to pass Codacy?

Comment (Abhi2424shek) [inline app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java]: Logging it would be nice.

**Comment by mayank408** [comment]
@the-dagger @jig08 There is no need for this as the app is not live on the playstore.

**Comment by Abhi2424shek** [comment]
@rohanagarwal94 any updates? It's been long! :)

**Comment by mariobehling** [comment]
Currently this is out of scope.

**Pull request closed without merging**